domains/rntxpurval.is-a-fullstack.dev.json

```diff
@@ -1,15 +1,15 @@
 {
     "description": "For Hosting My projects",
     "domain": "is-a-fullstack.dev",
-    "subdomain": "rntxpurval",
+    "subdomain": "tomato.rntxpurval",
```

Comment:
If you also change the name of the subdomain name, you need to change the name of the file.

Reply:
OK I will do it